### ISSUE TITLE
Gracefully handle missing accessibility data

### ISSUE THREAD

**Issue by avaleske** [CityOfDetroit/bloom#231]
We need the listings view to gracefully handle missing accessibility data. See #228 for the strategy for this, which could be either a simple "Unknown" or a prompt to contact the property manager.

Per the PRD:
Workflow for No Data: Contact Property via phone/email solution for no data / limited data sets (including accessibility)

**Comment by willrlin** [comment]
As part of this task, did we want to specifically show the a11y section but have the "contact property management" text there? I think right now we're just not showing the section at all. @tomgebauer what do you think?

**Comment by tomgebauer** [comment]
@willrlin I like this suggestion. TLDR; always show a11y section–if there is no specification we render "Contact property management"
